### PR TITLE
Fixed a few typos and language mistakes

### PR DESCRIPTION
**Pull request by Lemongrass3110**
* **Addressed Issue(s)**: None

* **Server Mode**: Renewal

* **Description of Pull Request**: 
@Everade brought the delayed typo to my attention so I flew over the script once again.

### CODE REVIEW

**Comment by aleos89** [comment]
I tried to make more sense of some lines. If I missed something let me know!

File: npc/airports/airships.txt [hunk 1]
```diff
@@ -452,7 +452,7 @@ airplane,221,158,2	script	Pilot#airplane	852,{
 		emotion ET_HUK;
 		mes "[Pilot]";
 		mes "Still, they where real";
```
Comment (zackdreaver) [inline npc/airports/airships.txt]: "they were" ?

File: npc/re/merchants/bio4_reward.txt [hunk 2]
```diff
@@ -30,7 +30,7 @@ lighthalzen,342,291,4	script	Weird old man#Bio4Reward	4_M_EINOLD,{
 	switch(rand(1,2)) {
 	case 1:
 		mes "[Weird old man]";
-		mes "I was on the way to visit my son who lives in Rune Midgarts. It's really far away, so I took the airplane, and..";
+		mes "I was on the way to visit my son who lives in Rune Midgard. It's really far away, so I took the airplane, and..";
 		next;
 		mes "[Weird old man]";
 		mes "those young people...so noisy.... I was standing but non of them offered me a seat..and they were just talking in loud voices which bothered everyone.";
```
Comment (zackdreaver) [inline npc/re/merchants/bio4_reward.txt]: "I was standing but **none** of them"

File: npc/re/merchants/bio4_reward.txt [hunk 3]
```diff
@@ -50,7 +50,7 @@ lighthalzen,342,291,4	script	Weird old man#Bio4Reward	4_M_EINOLD,{
 		break;
 	case 2:
 		mes "[Weird old man]";
-		mes "I was on the way to visit my son who lives in Rune Midgarts. It's really far away, so I took the airplane, and..";
+		mes "I was on the way to visit my son who lives in Rune Midgard. It's really far away, so I took the airplane, and..";
 		next;
 		mes "[Weird old man]";
 		mes "those young people...so noisy.... I was standing but non of them offered me a seat..and they were just talking in loud voices which bothered everyone.";
```
Comment (zackdreaver) [inline npc/re/merchants/bio4_reward.txt]: "I was standing but **none** of them"

File: npc/re/merchants/bio4_reward.txt [hunk 4]
```diff
@@ -194,7 +194,7 @@ lighthalzen,342,291,4	script	Weird old man#Bio4Reward	4_M_EINOLD,{
 					break;
 				case 5:
 					mes "<<Salvage Cape[0]>>";
-					mes "Allow to use 'Cicada Skin Shed'(Lv 1). Cool time for this skill will be 30 seconds. When equiping it, SP consumption for using any skill will be doubled, When taking it off, SP - 1200.";
+					mes "Allow to use 'Cicada Skin Shed'(Lv 1). Cool time for this skill will be 30 seconds. When equipping it, SP consumption for using any skill will be doubled, When taking it off, SP - 1200.";
```
Comment (zackdreaver) [inline npc/re/merchants/bio4_reward.txt]: "Cool time" > "Cool down time" ?

**Pull request merged**